### ISSUE TITLE
Incorrect Results for Pmin and Pmax even though state space is correct

### ISSUE THREAD

**Issue by ifndefJOSH** [fluentverification/stamina-storm#10]
The current property modification code for some reason does not directly work. The values of P_min and P_max are incorrect even though the state space is properly truncated.

**Comment by ifndefJOSH** [comment]
Attempting to write modified files to do this but the model created and the modified properties for some reason cause the truncation to break again.

See commit 936e30efa95f3de7aa7775aeaf82d84f79527ad6

**Comment by ifndefJOSH** [comment]
Proof of the above in commit 86db9c56a7665d728b1393d8273e3932dec52344

**Comment by ifndefJOSH** [comment]
Ok--the issue is not with the property modification. Compare the outputs of commits:
86f6b0786f9bb9390bf989f498291fc5916cceef (Latest one with different property modification)
AND
7a498e9a53aa8a88d2db61b16583b0e6245ad716 (Original property modification)

Keeping the newer one because it supports more than `true U`, whereas the older one only supported `true U`.

**Comment by ifndefJOSH** [comment]
Autocheck script: (assumes that `MODEL_FILE` and `PROPERTIES_FILE` are declared)
```bash
mkdir tmp
git checkout 7a498e9a53aa8a88d2db61b16583b0e6245ad716
cd build
cmake .. && make -j8
# Kill after 7 minutes
seq 1 | xargs -II timeout $((7 * 60)) ./stamina-cplusplus $MODEL_FILE $PROPERTIES_FILE 2>>tmp/original.txt
git checkout 86f6b0786f9bb9390bf989f498291fc5916cceef
cmake .. && make -j8
# Kill after 7 minutes
seq 1 | xargs -II timeout $((7 * 60)) ./stamina-cplusplus $MODEL_FILE $PROPERTIES_FILE 2>>tmp/latest.txt
diff original.txt latest.txt
```

**Comment by ifndefJOSH** [comment]
Script updated in f00a70f629dea8e8712f281137ef17d7e6956f75

**Comment by ifndefJOSH** [comment]
Found a potential problem:

- `currentRow` was set to 0, when it should have been 1
- No self loop was present on the absorbing state

Testing this fix in commit aa75ee1ac179bc8d29bbf28ef77298ce16207067

The results of this test showed that it did not fix the problem

**Comment by ifndefJOSH** [comment]
I found what could be a problem: the `currentRow` wasn't actually reflecting the state index, although it should.

Fix in commit 83f7e3403447313ce22859b32870a44eb39809d5

This didn't fix this but got it a lot closer.

**Comment by ifndefJOSH** [comment]
According to Dr Zhang, the results were actually correct
